sdk/containerregistry/azure-containerregistry/samples/load_registry.py

```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+
```

Comment:
Looks like this file name should be prefixed with `sample_`?
Also - there doesn't seem to be an async version.